server/README.md

```diff
@@ -27,6 +27,9 @@ The FASTEN-server is a component necessary for running [FASTEN-specific plugins]
     - `-pl` `--plugin_list` List of plugins to run. Can be used multiple times.
     - `-po` `--plugin_output` Path to directory where plugin output messages will be stored
     - `-pol` `--plugin_output_link` HTTP link to the root directory where output messages will be stored
+- `cg` `--consumer_group` Name of the consumer group. Defaults to (canonical) name of the plugin.
```

Comment:
Why `--consumer_group` argument for the server?

Reply:
Because it was not configurable before and always set to the name of the class. Therefore I couldn't deploy OPAL again for a test run without using a different consumer groud. It would use the same consumer group as used for production + the same offsets (which I want to prevent for a test run). Btw this change was already implemented in [this PR](https://github.com/fasten-project/fasten/pull/183).